src/main/java/use_case/compute_time/ComputeTimeOutputData.java

```diff
@@ -0,0 +1,43 @@
+package use_case.compute_time;
+
+import entity.AttractionData;
+import java.util.List;
+
+/**
+ * Represents the output data of the "Compute Time" use case.
+ * This class encapsulates a list of sequentially ordered locations and a flag indicating
+ * whether the use case failed.
+ */
+public class ComputeTimeOutputData {
+    private final List<AttractionData> sequentialLocations;
+    private final boolean useCaseFailed;
```

Comment:
Nit: I could be wrong, but since most errors are handled on the same page, we probably don't need this variable?